ntbs-service/DataMigration/NotificationSearcher.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using ntbs_service.Models;
+using Dapper;
+using Microsoft.Extensions.Configuration;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.DataMigration
+{
+    public interface INotificationSearcher

```

Comment:
Does this stillw ant to be called the searcher? I doubt that, as we are implementing a migration/import.